app/models/sponsorship.rb

```diff
@@ -30,23 +30,12 @@ class Sponsorship < ActiveRecord::Base
 
 private
 
-  def start_date_no_later_than_1st_of_next_month
```

Comment:
@b264 do you remember the utility of this private method ? I didn't find any references through the source code :thinking: